l10n_it_account_stamp/models/invoice.py

```diff
@@ -7,6 +7,9 @@ class AccountInvoice(models.Model):
     _inherit = 'account.invoice'
     tax_stamp = fields.Boolean(
         "Tax Stamp", readonly=True, states={'draft': [('readonly', False)]})
+    auto_compute_stamp = fields.Boolean(
+        related='company_id.tax_stamp_product_id.auto_compute',
+        readonly=True)
```

Comment:
Since `12.0` version the related fields are readonly by default. So you could remove `readonly=True`.